Automattic/wp-calypso
# Partner Portal: Reorganize code in preparation for the data layer implementation

#### Changes proposed in this Pull Request

* Reorganize Partner Portal code to allow for better data modularization and consistent code style.

#### Testing instructions

* If you do not have a partner key, please contact Infinity for one or you will be unable to view the UI.
* Checkout PR locally, run `yarn && yarn start-jetpack-cloud`.
* Visit http://jetpack.cloud.localhost:3000/partner-portal and select your partner key, if prompted.
* You should be presented with a list of 4 placeholder licenses. There should be no visual differences between this branch and `trunk`.

![Screenshot 2021-02-10 at 17 24 03](https://user-images.githubusercontent.com/22746396/107530676-c8b50280-6bc4-11eb-8094-7de894c7edd9.png)

Part of #49344
